latex/interni/verbali/verbale_interno_23-11-22/res/sections/02_content.tex

```diff
@@ -1,24 +1,24 @@
 
-\subsection{Considerazioni sull'Issue tracking system\textsubscript{g}}
+\subsection{Considerazioni sull'Issue\textsubscript{g} tracking system}
 Abbiamo deciso di abbandonare l'utilizzo di Jira\textsubscript{g} a supporto di Github\textsubscript{g} in quanto l'automazione del sistema non sembra essere facilmente concretizzabile e rischia di creare una discrepanza del tracciamento delle issue\textsubscript{g} nei due sistemi
 
-\subsection{Inizio esplorazione dei framework}
-Interrogandoci sulla possibile aggiunta di nuove tecnologie da inserire nella requirement e technology baseline, abbiamo considerato i consigli del docente Cardin e abbiamo deciso di iniziare a esplorare l'utilizzo di framework per lo sviluppo in Javascript. Al momento Tommaso e Rino si occuperanno di dare un occhiata a Angular, React o Svelte.
+\subsection{Inizio esplorazione dei framework\textsubscript{g}}
+Interrogandoci sulla possibile aggiunta di nuove tecnologie da inserire nella requirement e technology baseline, abbiamo considerato i consigli del docente Cardin e abbiamo deciso di iniziare a esplorare l'utilizzo di framework\textsubscript{g} per lo sviluppo in Javascript. Al momento Tommaso e Rino si occuperanno di dare un occhiata a Angular, React o Svelte.
 
 
 \subsection{Documenti da aggiornare}
 Abbiamo deciso di proseguire nello sviluppo di alcuni file nel seguente modo:
 \begin{itemize}
-	\item \textit{analisi dei requisiti\textsubscript{g}}: il documento è a buon punto, ma va ancora rifinito e verificato. Come metodologia di verifica, oltre ad un controllo dei verificatori, abbiamo deciso di chiedere l'opinione del proponente; il file gli verrà inviato entro venerdì 25 novembre alle 12:00.
-	\item \textit{piano di progetto}: il documento va aggiornato inserendo un diagramma di Gantt, pianificazione dello sprint appena concluso, resoconto dello sprint appena concluso, e pianificazione del nuovo sprint
-	\item \textit{norme di progetto}: va eliminata la sezione dell'utilizzo di Jira e va aggiunta tra le norme di verifica, una sezione che dica che i verbali vanno approvati subito dopo la loro verifica
+	\item \textit{\textit{Analisi dei Requisiti}}: il documento è a buon punto, ma va ancora rifinito e verificato. Come metodologia di verifica\textsubscript{g}, oltre ad un controllo dei verificatori, abbiamo deciso di chiedere l'opinione del proponente\textsubscript{g}; il file gli verrà inviato entro venerdì 25 novembre alle 12:00.
+	\item \textit{\textit{Piano di Progetto}}: il documento va aggiornato inserendo un diagramma di Gantt\textsubscript{g}, pianificazione dello sprint\textsubscript{g} appena concluso, resoconto dello sprint\textsubscript{g} appena concluso, e pianificazione del nuovo sprint\textsubscript{g}
```

Comment:
Piano di Progetto era già in corsivo, di conseguenza è stato fatto un 'doppio' textit